src/ColumnDisplayFormatDialog.cpp

```diff
@@ -17,6 +17,7 @@ ColumnDisplayFormatDialog::ColumnDisplayFormatDialog(const QString& colname, QSt
     ui->comboDisplayFormat->addItem(tr("Hex number"), "hex");
     ui->comboDisplayFormat->addItem(tr("Octal number"), "octal");
     ui->comboDisplayFormat->addItem(tr("Round number"), "round");
+    ui->comboDisplayFormat->addItem(tr("Plist"), "plist");
```

Comment:
This will require some existence test for inclusion in DB4S, but don't know how to test for existence of a loaded SQLite function (maybe only executing it). If we include my PR  #1720, it wouldn't be really necessary, since users could enter a custom function as display format and save it to the project file.